### PR TITLE
add last_duration attribute to progress bar

### PR DESCRIPTION
**Pull request by mrocklin**
cc @jcrist

### CODE REVIEW

File: dask/diagnostics/progress.py [hunk 1]
```diff
@@ -93,6 +103,8 @@ def _finish(self, dsk, state, errored):
         sys.stdout.write('\n')
         sys.stdout.flush()
 
+        self.last_duration = elapsed
+
```
Comment (jcrist) [inline dask/diagnostics/progress.py]: If `elapsed < self._minimum` then this won't get set. Perhaps Move it above the first if statement.

Comment (mrocklin) [inline dask/diagnostics/progress.py]: Fixed

**Comment by mrocklin** [comment]
@jcrist ok to merge?

**Comment by jcrist** [comment]
+1

**Pull request merged**